src/test/java/org/jitsi/jicofo/MediaSourceGroupMapTest.java

```diff
@@ -30,137 +30,168 @@
 import static org.junit.Assert.*;
 
 /**
- * Tests for {@link MediaSSRCGroupMap}
+ * Tests for {@link MediaSourceGroupMap}
  *
  * @author Pawel Domas
  */
 @RunWith(JUnit4.class)
-public class MediaSSRCGroupMapTest
+public class MediaSourceGroupMapTest
 {
+    @Test
+    public void testContainsGroup()
+    {
+        /**
+         * Make sure that groups with the same semantic but different source
+         * types (rid vs ssrc) don't collide
```

Comment:
fixed (i left the other capitalization scheme for the variable/function names, but i agree this capitalization is appropriate for comments)